image/DecodedSurfaceProvider.cpp

```diff
@@ -113,6 +118,8 @@ size_t DecodedSurfaceProvider::LogicalSizeInBytes() const {
 
 void DecodedSurfaceProvider::Run() {
   MutexAutoLock lock(mMutex);
+  recordreplay::RecordReplayAssert("DecodedSurfaceProvider::Run %u",
+    (unsigned) recordreplay::ThingIndex(this));
```

Comment:
These assertions should have a comment referencing the issue.

Reply:
Thanks for pointing that out.  Will do.